Pyjion/pyjit.cpp

```diff
@@ -21,12 +21,17 @@
 * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 * OTHER DEALINGS IN THE SOFTWARE.
 *
+
+
 */
 
 #include "pyjit.h"
 #include "pycomp.h"
 
 #include <vector>
+//#define NO_TRACE
```

Comment:
I like keeping these here just to flip them on and off as we may experiment with different tracing strategies.